<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdvancedSettingsDialog</class>
 <widget class="QDialog" name="AdvancedSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Advanced Settings</string>
  </property>
  <widget class="QLabel" name="adv_general_settings_label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>90</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Advanced General Settings</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="acq_order_combo_box">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>150</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Sets acquisition priority order.&lt;/p&gt;&lt;p&gt;If set to TIME_SAMP, the outermost loop will be the time points loop. All fish will be imaged in order, and then the program will wait for the next time point. This is the default acquisition mode.&lt;/p&gt;&lt;p&gt;If set to SAMP_TIME, the outermost loop will be the fish number loop. The first fish will be imaged, then the program will wait for the next time point, then the same fish will be imaged, and so on. Only once all time points are complete will the program move on to the next fish, for which it will complete another full time series.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
  </widget>
  <widget class="QLabel" name="z_stack_exp_unit_label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>140</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ms</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="z_stack_settings_label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Z-Stack Settings</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="z_stack_exposure_line_edit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>140</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Exposure time for use in Z-stack. If the Cycle Channels box is not checked, this exposure time must be less than or equal to 1/(stage speed) seconds. That is, if stage speed = 30, then the exposure time must be less than or equal to 1/30 = 33 ms.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="lsrm_check_box">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>120</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If checked, enables light sheet readout mode while acquiring images taken in fluorescent channels. LSRM  should be configured in the Spim Controls GUI before using it in acquisitions.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Lighsheet Readout Mode</string>
   </property>
  </widget>
  <widget class="QLabel" name="z_stack_exposure_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Z-Stack Exposure Time:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="video_spectral_check_box">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>40</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If checked, Video acquisitions will change filters every frame in the order specified by the Channel Order above. Note that this significantly slows down video acquisitions since switching between channels takes time and is only meant to be used if timing between channel frames is very important.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Spectral Video?</string>
   </property>
  </widget>
  <widget class="QLabel" name="video_settings_label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Video Settings</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="z_stack_spectral_check_box">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If checked, Z-stack acquisitions will change filters every frame in the order specified by the Channel Order above. The stage will move by the amount specified by Stepsize, take an image with each channel, move by the Stepsize again, take images in all channels, and so on. &lt;/p&gt;&lt;p&gt;Note that this is very fringe and should only be used if images of the same plane with different channels is time-sensitive for some reason.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Spectral Z-Stack?</string>
   </property>
  </widget>
  <widget class="QLabel" name="stage_speed_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Stage Speed:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="stage_speed_combo_box">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Sets stage speed to be used during continuous Z-stack. Note that this value is irrelevant if Spectral Scan is checked above.&lt;/p&gt;&lt;p&gt;Since 30 is the full sensor maximum framerate of the PCO camera, 30 um/s is the current default speed. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>-50</x>
     <y>180</y>
     <width>531</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>4</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>-140</y>
     <width>20</width>
     <height>331</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>4</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="edge_trigger_check_box">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If checked, Z-stack acquisitions will change filters every frame in the order specified by the Channel Order above. The stage will move by the amount specified by Stepsize, take an image with each channel, move by the Stepsize again, take images in all channels, and so on. &lt;/p&gt;&lt;p&gt;Note that this is very fringe and should only be used if images of the same plane with different channels is time-sensitive for some reason.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Edge Trigger?</string>
   </property>
  </widget>
  <widget class="QLabel" name="acq_order_label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>150</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Acquisition Order:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>70</y>
     <width>531</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>4</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="save_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>260</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Second Save Path:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="second_save_path_browse_button">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Browse and choose directory for acquisition to be saved&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Browse...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="second_save_path_line_edit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>260</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Save location of acquisition&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="second_save_path_check_box">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>200</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If checked, Video acquisitions will change filters every frame in the order specified by the Channel Order above. Note that this significantly slows down video acquisitions since switching between channels takes time and is only meant to be used if timing between channel frames is very important.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Second Save Path</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
